types/chrome/index.d.ts

```diff
@@ -7235,7 +7235,9 @@ declare namespace chrome.runtime {
         icons?: ManifestIcons | undefined;
 
         // Optional
-        author?: string | undefined;
+        author?: {
+            email: string;
+        } | undefined;
```

Comment:
Can you add a comment like this, from https://developer.chrome.com/docs/extensions/reference/manifest/author? 

```suggestion
        author?: {
            // This is the email address of the extension author. When 
            // publishing a CRX file to the Chrome Web Store, this string MUST
            // match the email address of the account used to publish the
            // extension.
            email: string;
        } | undefined;
```

Reply:
I think the precedent with manifest is to leave out the comments. We're not setting the standard for the actual .json file, we can't get that intellisense yet. I don't see a point in adding a comment here when authors are already  looking at chrome docs for the manifest.

Reply:
That's fair, but there are folks who generate their JSON file through TypeScript (I do this weirdly enough). I'm OK with either but still generally prefer to have some documentation.